src/Chapter4.hs

```diff
@@ -612,6 +639,20 @@ Implement the 'Monad' instance for our lists.
 -}
 
 
+instance Monad List where
+  (>>=) :: List a -> (a->List b)->List b
```

Comment:
In Haskell all functions and arguments are always separated by spaces, even in types 👾 

```suggestion
  (>>=) :: List a -> (a -> List b) -> List b
```